tests/tasks/setup_802_1x_server.yml

```diff
@@ -1,5 +1,13 @@
 # SPDX-License-Identifier: BSD-3-Clause
 ---
+# This task can be removed once the RHEL-8.5 is not tested anymore
+- name: Install hostapd via CentOS Stream
+  command: dnf -y install http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/hostapd-2.10-1.el8.x86_64.rpm
+  when:
+    - ansible_distribution_version | float < 8.6 and
```

Comment:
for the sake of consistency, you should not mix using an explicit `and`, and using the list format - so
```yaml
  when:
   - ansible_distribution_version | float < 8.6
   - ansible_distribution_major_version == '8'
   - ansible_distribution == 'RedHat'
```

Reply:
Thanks for notifying me on this. Updated.